packages/virtual-list/src/vaadin-virtual-list.d.ts

```diff
@@ -33,14 +32,32 @@ export { VirtualListDefaultItem, VirtualListItemModel, VirtualListRenderer };
  *
  * Attribute        | Description
  * -----------------|--------------------------------------------
- * `overflow`       | Set to `top`, `bottom`, both, or none.
+ * `overflow`       | Set to `top`, `bottom`, both, or none
+ * `interacting`    | Keyboard navigation in interaction mode
+ * `navigating`     | Keyboard navigation in navigation mode
+ * `selected`       | Set on a child element when the item is selected
+ * `focused`        | Set on a child element when the item is focused
  *
  * See [Virtual List](https://vaadin.com/docs/latest/components/virtual-list) documentation.
+ *
+ * @fires {CustomEvent} selected-items-changed - Fired when the `selectedItems` property changes.
  */
 declare class VirtualList<TItem = VirtualListDefaultItem> extends ThemableMixin(ElementMixin(HTMLElement)) {}
 
 // eslint-disable-next-line @typescript-eslint/no-empty-object-type
-interface VirtualList<TItem = VirtualListDefaultItem> extends VirtualListMixinClass<TItem> {}
```

Comment:
The ESLint comment above can be probably removed since the interface is no longer empty.